src/azul/service/manifest_service.py

```diff
@@ -175,7 +187,9 @@ def _derive_manifest_key(self, filters: Filters) -> str:
         git_commit = config.lambda_git_status['commit']
         manifest_namespace = uuid.UUID('ca1df635-b42c-4671-9322-b0a7209f0235')
         filter_string = repr(sort_frozen(freeze(filters)))
-        return str(uuid.uuid5(manifest_namespace, git_commit + filter_string))
+        content_hash = str(content_hash)
+        assert all(',' not in param for param in [git_commit, format_, content_hash])
```

Comment:
```suggestion
        assert not any(',' in param for param in [git_commit, format_, content_hash])
```

reads better

Reply:
```suggestion
        assert all(',' not in param for param in (git_commit, format_, content_hash))
```

since you're passing a tuple to `.join` below.